internal/prompt/prompt.go

```diff
@@ -92,3 +93,15 @@ func SelectInput(name string, message string, help string, options []string, def
 
 	return input
 }
+
+func Password(inputs string) string {
```

Comment:
Can you please have it return an error and then handle the error? Like with `Ask`, `AskOne` and `AskBool`.